### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
### Landing Page
I have converted my home page screen to react, created a separate component for navigation and footer. Also, added BrowserRoutes and have implemented routes to various pages in the app. 

Other Changes
- Added Car-Smart Favicon
- Imported self-made Component Library from [ Odyssey UI](https://odysseyui.netlify.app/)
- Sourced all images from online CDN hosting.

Files to check
-  src/App.js
- src/Router/Router.jsx
- src/components/Footer/Footer.js
- src/components/Navigation/Navigation.jsx
- src/pages/Home/Home.jsx

PS- Ignore other files
- All CSS Files have been reviewed earlier in this repositiory [Car-Smart](https://github.com/Prabhav-Sharma/Car-Smart)

### CODE REVIEW

File: src/components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.nav-title:hover {
+    flex-direction: row-reverse;
+    cursor: pointer;
+}
+
+.nav-title:first-child:hover {
+    transform: translateX(5px);
+}
+
+input {
```
Comment (SukanyaSen530) [inline src/components/Navigation/navigation.css]: Suggestion - use a class to change it, if it is not a global style

**Review by SukanyaSen530 — COMMENTED** [review]
PR is descriptive. Suggestion - Use classes to change the html elements.

**Review by Richa-rani02 — APPROVED** [review]

**Review by KanekarSnehal — APPROVED** [review]
- Code looks well structured.
-  In Home.jsx you can make separate component for repetitive code.

File: src/components/Navigation/navigation.css (hunk 1 above)
Comment (Prabhav-Sharma) [reply src/components/Navigation/navigation.css]: I've made the suggested changes, thank you!

**Comment by Prabhav-Sharma** [comment]
> 
I've further broken the repetitive code down into components, Thank you for the suggestion.

**Pull request merged**